### DISCUSSION TITLE
Weather station

### DISCUSSION THREAD

**Discussion by elik745i** [arendst/Tasmota#10607] [Show and tell]
![20210113_150407](https://user-images.githubusercontent.com/17794736/104853895-4c106a80-591d-11eb-921d-479649e283cc.jpg)

Guys I was working whole week on my weather station. Finally it is ready and installed today outside for testing. All about project you can read here: https://3dtoday.ru/blogs/elik745i/moya-pogodnaya-stanciya
STL files here: https://www.thingiverse.com/thing:4726516
Video of design and work here: https://youtu.be/mxx-0kPgwpM

I'm really satisfied with results and how tasmota firmware is working on it, I had to write my first driver for digital compass for it and with help from community I succeed!

Firmware here: [firmware.zip](https://github.com/arendst/Tasmota/files/5826673/firmware.zip)
Config: 
![config_tasmota](https://user-images.githubusercontent.com/17794736/104853923-76fabe80-591d-11eb-8d24-40bcb388e6e5.png)
Schematics: 
![dwg](https://user-images.githubusercontent.com/17794736/104853951-a14c7c00-591d-11eb-80d5-c85cab286e3b.png)

Final screen in OPENHAB2.5:
![Final](https://user-images.githubusercontent.com/17794736/104853961-b2958880-591d-11eb-9e3e-a341b25f8303.png)
![20210117_133356](https://user-images.githubusercontent.com/17794736/104853968-bcb78700-591d-11eb-9ba6-bc173096ae75.jpg)

Thanks to all, who helped me with this small project.

**Comment by TrDA-hab** [comment]
Cool project. I really envy you !!!
Thanks for sharing!
And be sure to add a link to discuss the driver (not for download!) For the HCM5883L. It was unusual, I love it!
